packages/apputils-extension/src/index.ts

```diff
@@ -611,10 +611,34 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
  * The default HTML sanitizer.
  */
 const sanitizer: JupyterFrontEndPlugin<ISanitizer> = {
-  id: '@jupyter/apputils-extension:sanitizer',
+  id: '@jupyterlab/apputils-extension:sanitizer',
```

Comment:
While it is fixing what seems to be an unintentional typo (which was there since this plugin was introduced in https://github.com/jupyterlab/jupyterlab/pull/9873), it is a potentially breaking change. We may need to target 4.0 because of that.

I checked for other occurrences of this issue, and only found one more in tests: https://github.com/jupyterlab/jupyterlab/blob/83b12ea7359a1f547585b2ea2b16b48ad0c10e27/packages/notebook/test/widget.spec.ts#L161.

Reply:
Okay fair enough. Is there anything else I need to do on my end?

Reply:
In my opinion we should correct the id name on master (aka for 4.0.0) and not do it when backporting this PR.